dataactvalidator/validation_handlers/errorInterface.py

```diff
@@ -6,88 +6,80 @@
 from dataactcore.models.lookups import ERROR_TYPE_DICT
 
 
-class ErrorInterface:
-    """Manages communication with error database."""
+def record_row_error(error_list, job_id, filename, field_name, error_type, row, original_label=None, file_type_id=None,
+                     target_file_id=None, severity_id=None):
+    """ Add this error to running sum of error types
 
-    def __init__(self):
-        """ Create empty row error dict """
-        self.rowErrors = {}
+    Args:
+        job_id: ID of job in job tracker
+        filename: name of error report in S3
+        field_name: name of field where error occurred
+        error_type: type of error, value will be mapped to ValidationError class,
+            for rule failures this will hold entire message
+        row: Row number error occurred on
+        original_label: Label of rule
+        file_type_id: Id of source file type
+        target_file_id: Id of target file type
+        severity_id: Id of error severity
+    """
```

Comment:
Where is the `error_list` argument in this args list?

This function also returns something now, where is that in the comment?